log/gokit.go

```diff
@@ -6,101 +6,68 @@ package log
 
 import (
 	"fmt"
+	"io"
 	"os"
 
 	"github.com/go-kit/log"
 	"github.com/go-kit/log/level"
 )
 
-// NewGoKitFormat creates a new Interface backed by a GoKit logger
-// format can be "json" or defaults to logfmt
-func NewGoKitFormat(l Level, f Format) Interface {
-	var logger log.Logger
-	if f.s == "json" {
-		logger = log.NewJSONLogger(log.NewSyncWriter(os.Stderr))
-	} else {
-		logger = log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr))
-	}
-	return addStandardFields(logger, l)
+const (
+	LogfmtFormat = "logfmt"
+	JSONFormat   = "json"
+)
+
+// NewGoKit creates a new GoKit logger with the given format.
+// If the given format is nil, logfmt is used.
```

Comment:
[nit] Technically a string can't be nil.

```suggestion
// If the given format is empty or unknow, logfmt is used.
```